src/build/template.html

```diff
@@ -15,8 +15,20 @@
        https://developers.google.com/web/fundamentals/native-hardware/fullscreen/ -->
   <meta name="mobile-web-app-capable" content="yes" >
   <meta name="apple-mobile-web-app-capable" content="yes">
-  <meta name="apple-mobile-web-app-title" content="{intl.appName}" >
-  <meta name="apple-mobile-web-app-status-bar-style" content="default" >
+  <meta name="apple-mobile-web-app-title" content="{intl.appName}">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
```

Comment:
Does `black-translucent` play well with dark mode vs light mode? Or only in one or the other?

Reply:
Yes it works in both scenarios, someone in apple decided to name it black-translucent. It's actually translucent.

Reply:
Yes, the service worker will eagerly download all images when the site is first loaded. We should add a filter here:

https://github.com/nolanlawson/pinafore/blob/5ecf8b8ab959698c64432eda785b0f2774576635/src/service-worker.js#L29-L36

For instance, we could add a test for the string "splash".